### PR TITLE
some cleaning

### PR DESCRIPTION
**Pull request by LunarcatOwO**
Did some formating,
Also removed 2 imports from one file as they were not being used!

**NOT SURE IF IT WORKS, PLEASE TEST BEFORE MERGING**

### CODE REVIEW

File: src/main/java/me/iseal/powergems/gems/IronGem.java [hunk 1]
```diff
@@ -20,51 +20,56 @@
...
-            l.warning("[PowerGems] "+plr.getDisplayName()+" used Iron Gem Shift while already having the modifiers, please report this to the developer");
+            l.warning("[PowerGems] " + plr.getDisplayName()
+                    + " used Iron Gem Shift while already having the modifiers, please report this to the developer");
         }
         plr.setVelocity(new Vector(0, 0, 0));
         Bukkit.getScheduler().runTaskLater(Main.getPlugin(), () -> {
             OfflinePlayer op = Bukkit.getOfflinePlayer(plr.getUniqueId());
-            if (op.isOnline()) {;
+            if (op.isOnline()) {
+                ;
```
Comment (iseal56) [inline src/main/java/me/iseal/powergems/gems/IronGem.java:51]: Why is this here?

File: src/main/java/me/iseal/powergems/listeners/inventoryCloseListener.java [hunk 2]
```diff
@@ -21,34 +21,34 @@ public class inventoryCloseListener implements Listener {
...
+    public void onClose(InventoryCloseEvent e) {
         if (!(e.getInventory().getHolder() instanceof Player plr)) {
             return;
         }
         if (!(e.getView().getBottomInventory() instanceof PlayerInventory pi)) {
             return;
         }
-        //checkIfMultipleGems(plr);
-        if (!e.getView().getBottomInventory().containsAtLeast(randomGem, 1)){
+        // checkIfMultipleGems(plr);
```
Comment (iseal56) [inline src/main/java/me/iseal/powergems/listeners/inventoryCloseListener.java:31]: Its a commented function call, don't add the space

File: src/main/java/me/iseal/powergems/listeners/passivePowerListeners/damageListener.java [hunk 3]
```diff
@@ -15,19 +15,22 @@
...
-    private void checkIfFall(Player p, EntityDamageEvent event){
-        if (!event.getCause().equals(EntityDamageEvent.DamageCause.FALL)) return;
-        for (ItemStack i : gm.getPlayerGems(p)){
-            if (noFall.contains(i.getItemMeta().getPersistentDataContainer().get(Main.getGemPowerKey(), PersistentDataType.STRING))){
+    private void checkIfFall(Player p, EntityDamageEvent event) {
+        if (!event.getCause().equals(EntityDamageEvent.DamageCause.FALL))
+            return;
+        for (ItemStack i : gm.getPlayerGems(p)) {
+            if (noFall.contains(i.getItemMeta().getPersistentDataContainer().get(Main.getGemPowerKey(),
+                    PersistentDataType.STRING))) {
```
Comment (iseal56) [inline src/main/java/me/iseal/powergems/listeners/passivePowerListeners/damageListener.java:33]: Keep this as it was. It's the type of the object

File: src/main/java/me/iseal/powergems/listeners/useEvent.java [hunk 4]
```diff
@@ -44,16 +43,18 @@ public void onUse(PlayerInteractEvent e) {
         }
         ItemMeta meta = item.getItemMeta();
         PersistentDataContainer dataContainer = meta.getPersistentDataContainer();
-        if (dataContainer.has(Main.getIsGemKey(), PersistentDataType.BOOLEAN) && dataContainer.has(Main.getGemPowerKey(), PersistentDataType.STRING)) {
+        if (dataContainer.has(Main.getIsGemKey(), PersistentDataType.BOOLEAN)
+                && dataContainer.has(Main.getGemPowerKey(), PersistentDataType.STRING)) {
             if (tdm.cantUseGems.containsKey(player)) {
                 if (System.currentTimeMillis() < tdm.cantUseGems.get(player)) {
-                    player.sendMessage(ChatColor.DARK_RED + "You can't use gems for another " + (tdm.cantUseGems.get(player) - System.currentTimeMillis()) / 1000 + " seconds!");
+                    player.sendMessage(ChatColor.DARK_RED + "You can't use gems for another "
```
Comment (iseal56) [inline src/main/java/me/iseal/powergems/listeners/useEvent.java:50]: Keep this as it was, it's clearer as the message is only 1 line in game

File: src/main/java/me/iseal/powergems/managers/CooldownManager.java [hunk 5]
```diff
@@ -21,21 +21,23 @@ public class CooldownManager {
     public void setRightClickCooldown(Player player, long time, Class<?> fromClass) {
         if (cm.isDragonEggHalfCooldown()) {
             if (player.getInventory().contains(Material.DRAGON_EGG)) {
-                rightClickCooldowns.add(new CooldownObject(player, fromClass, (time * 500) + System.currentTimeMillis()));
+                rightClickCooldowns
+                        .add(new CooldownObject(player, fromClass, (time * 500) + System.currentTimeMillis()));
```
Comment (iseal56) [inline src/main/java/me/iseal/powergems/managers/CooldownManager.java:25]: Keep this as it was, dont separate the object from the function call

File: src/main/java/me/iseal/powergems/managers/CooldownManager.java [hunk 6]
```diff
@@ -21,21 +21,23 @@ public class CooldownManager {
...
-        rightClickCooldowns.add(new CooldownObject(player, fromClass, (time*1000)+System.currentTimeMillis()));
+        rightClickCooldowns.add(new CooldownObject(player, fromClass, (time * 1000) + System.currentTimeMillis()));
     }
 
     public void setLeftClickCooldown(Player player, long time, Class<?> fromClass) {
         if (cm.isDragonEggHalfCooldown()) {
             if (player.getInventory().contains(Material.DRAGON_EGG)) {
-                leftClickCooldowns.add(new CooldownObject(player, fromClass, (time * 500) + System.currentTimeMillis()));
+                leftClickCooldowns
+                        .add(new CooldownObject(player, fromClass, (time * 500) + System.currentTimeMillis()));
```
Comment (iseal56) [inline src/main/java/me/iseal/powergems/managers/CooldownManager.java:36]: Same as before

File: src/main/java/me/iseal/powergems/managers/GemManager.java [hunk 7]
```diff
@@ -386,44 +432,53 @@ public String getName(ItemStack item) {
...
-        if (!isGem(item)) return null;
+    public Class<?> getGemClass(ItemStack item) {
+        if (!isGem(item))
+            return null;
         try {
-            return Class.forName("me.iseal.powergems.gems."+item.getItemMeta().getPersistentDataContainer().get(gemPowerKey, PersistentDataType.STRING)+"Gem");
-        } catch (Exception e){
+            return Class.forName("me.iseal.powergems.gems."
+                    + item.getItemMeta().getPersistentDataContainer().get(gemPowerKey, PersistentDataType.STRING)
+                    + "Gem");
```
Comment (iseal56) [inline src/main/java/me/iseal/powergems/managers/GemManager.java:446]: Move this up a line, keep the other newline

File: src/main/java/me/iseal/powergems/managers/RecipeManager.java [hunk 8]
```diff
@@ -155,35 +156,39 @@ private void upgradeRecipe(){
                     }
                     sr.setIngredient('g', new RecipeChoice.ExactChoice(oldStack));
                     Bukkit.getServer().addRecipe(sr);
-                /*sr.shape("nen","ege","nen");
-                sr.setIngredient('n', Material.NETHERITE_INGOT);
-                sr.setIngredient('e', Material.EXPERIENCE_BOTTLE);
-                sr.setIngredient('g', new RecipeChoice.ExactChoice(oldStack));
-                Bukkit.getServer().addRecipe(sr);*/
+                    /*
```
Comment (iseal56) [inline src/main/java/me/iseal/powergems/managers/RecipeManager.java:159]: Can remove this part safely

File: src/main/resources/plugin.yml [hunk 9]
```diff
@@ -1,7 +1,7 @@
 name: PowerGems
-version: '${project.version}'
+version: "${project.version}"
```
Comment (iseal56) [inline src/main/resources/plugin.yml:2]: Keep as before

File: src/main/resources/plugin.yml [hunk 10]
```diff
@@ -1,7 +1,7 @@
...
 main: me.iseal.powergems.Main
-api-version: '1.19'
+api-version: "1.19"
```
Comment (iseal56) [inline src/main/resources/plugin.yml:4]: Keep as before

File: src/main/resources/plugin.yml [hunk 11]
```diff
@@ -21,23 +21,23 @@ commands:
...
+  powergems.checkupdates:
+    description: Permission to use /checkupdates
+    default: true
+  powergems.reloadconfig:
+    description: Permission to use /reloadconfig
+    default: op
+  powergems.*:
+    description: Permission to use all PowerGems commands
+    default: op
+    children:
```
Comment (iseal56) [inline src/main/resources/plugin.yml:39]: Needs testing, might not work

**Review by iseal56 — CHANGES_REQUESTED** [review]
Generally good, needs some minor changes for clarity and some testing

File: src/main/resources/plugin.yml (hunk 11 above)
Comment (LunarcatOwO) [reply src/main/resources/plugin.yml:39]: Test it.

File: src/main/java/me/iseal/powergems/gems/IronGem.java [hunk 12]
```diff
@@ -47,8 +47,7 @@ protected void rightClick(Player plr) {
         plr.setVelocity(new Vector(0, 0, 0));
         Bukkit.getScheduler().runTaskLater(Main.getPlugin(), () -> {
             OfflinePlayer op = Bukkit.getOfflinePlayer(plr.getUniqueId());
-            if (op.isOnline()) {
-                ;
+            if (op.isOnline()) {;
```
Comment (iseal56) [inline src/main/java/me/iseal/powergems/gems/IronGem.java:50]: Remove the semicolon. its useless

File: src/main/java/me/iseal/powergems/listeners/passivePowerListeners/damageListener.java [hunk 13]
```diff
@@ -25,12 +25,10 @@ public void onDamage(EntityDamageEvent e) {
...
     }
 
-    private void checkIfFall(Player p, EntityDamageEvent event) {
-        if (!event.getCause().equals(EntityDamageEvent.DamageCause.FALL))
-            return;
-        for (ItemStack i : gm.getPlayerGems(p)) {
-            if (noFall.contains(i.getItemMeta().getPersistentDataContainer().get(Main.getGemPowerKey(),
-                    PersistentDataType.STRING))) {
+    private void checkIfFall(Player p, EntityDamageEvent event){
+        if (!event.getCause().equals(EntityDamageEvent.DamageCause.FALL)) return;
```
Comment (iseal56) [inline src/main/java/me/iseal/powergems/listeners/passivePowerListeners/damageListener.java:29]: This line can stay as before

Comment (iseal56) [reply src/main/java/me/iseal/powergems/listeners/passivePowerListeners/damageListener.java:29]: So only the return statement

**Review by iseal56 — CHANGES_REQUESTED** [review]
Almost done, just need a couple of changes

**Review by iseal56 — APPROVED** [review]
Every issue fixed

**Pull request merged**